arbos/programs/cgo_test.go

```diff
@@ -7,12 +7,16 @@
 package programs
 
 import (
+	"flag"
 	"fmt"
-	"os"
 	"strings"
 	"testing"
 )
 
+var (
+	compileFlag = flag.String("TEST_COMPILE", "", "[STORE|LOAD] to allow store/load in compile test")
```

Comment:
By convention, I think we almost always use lowercase flag names separated by underbars.
Can we call this `test_compile` and change the others to be that way as well?

Reply:
Done.